<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGGEditDuctParameters</class>
 <widget class="QWidget" name="RGGEditDuctParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>405</width>
        <height>653</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Duct Segments:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTableWidget" name="ductSegmentTable">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>175</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="segSplit">
              <property name="text">
               <string>Split</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="segDeleteUp">
              <property name="text">
               <string>Delete Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="segDeleteDown">
              <property name="text">
               <string>Delete Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="materialLayerStack">
            <widget class="QWidget" name="page_3"/>
            <widget class="QWidget" name="page_4"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QGroupBox" name="materialBox">
         <property name="title">
          <string>Materials Layers:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="addMaterialLayerBefore">
              <property name="text">
               <string>Add Before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addMaterialLayerAfter">
              <property name="text">
               <string>Add After</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="deleteMaterialLayer">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="isCrossSectionButton">
              <property name="text">
               <string>Cross Section</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="isHexButton">
              <property name="text">
               <string>Hex</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="infoButton">
              <property name="text">
               <string>info</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dlLabel">
         <property name="text">
          <string>Duct Length:      </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="ductLengthLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Z1:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="z1LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Z2:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="z2LineEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ductName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pdLabel">
         <property name="text">
          <string>Duct Pitch:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="ductPitchLayout">
         <item>
          <widget class="QLabel" name="ductPitchXLabel">
           <property name="text">
            <string>X:  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ductPitchX"/>
         </item>
         <item>
          <widget class="QLabel" name="ductPitchYLabel">
           <property name="text">
            <string>Y:  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ductPitchY"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ductName</tabstop>
  <tabstop>ductSegmentTable</tabstop>
  <tabstop>segSplit</tabstop>
  <tabstop>segDeleteUp</tabstop>
  <tabstop>segDeleteDown</tabstop>
  <tabstop>addMaterialLayerBefore</tabstop>
  <tabstop>addMaterialLayerAfter</tabstop>
  <tabstop>deleteMaterialLayer</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
